package.json

```diff
@@ -15,13 +15,15 @@
   },
   "scripts": {
     "build": "ember build",
+    "install": "cd node_modules/comlink && npm install && npm run build",
```

Comment:
Does comlink need to be compiled? Does this have any negatives or gotchas?

Reply:
Yeah, the compile step is necessary when using the git repo source. Comlink’s written in TypeScript, and gets compiled to JS when published to npm, so here we need to do it on our own.

It’s just a temporary solution for sharing this work-in-progress. I wouldn’t even try publishing ECFT with it. 😅

Reply:
Ah cool, good to know